### ISSUE TITLE
Bug: Not all image URLs are supported

### ISSUE THREAD

**Issue by Anush008** [qdrant/qdrant-web-ui#253]
The web UI looks for image URLs ending with `jpg|jpeg|png|webp|gif|svg` before displaying them on the dashboard.

So it won't work for ones like
https://cdn.discordapp.com/attachments/927784326899331092/1301524272585379932/og.png?ex=6724ca6f&is=672378ef&hm=6584c97a3a83eb4e23505bfdb0b88739c36b494deb9501c60ae9d9bcb652601e&

I think we should check if the value is a URL and image content type is returned in its HEAD response.

Related file:
https://github.com/qdrant/qdrant-web-ui/blob/296764642e8ebefcffb8b08d7d69246dafccf9e1/src/components/Points/PointImage.jsx#L10-L21

**Comment by kartik-gupta-ij** [comment]
Hey @Anush008,

Thanks for pointing this out! Currently, the Web UI indeed looks for image URLs containing extensions like .jpg|jpeg|png|webp|gif|svg within the string, not just at the end. This means URLs like the one you shared should be rendered fine, as long as they include one of these extensions.
<img width="1710" alt="Screenshot 2024-11-01 at 5 08 29 PM" src="https://github.com/user-attachments/assets/fe7facf6-5ad6-4f37-9d73-11403a09cd36">


Regarding the idea of checking the content type via a HEAD request to confirm it's an image – that’s a solid approach, but it might run into CORS issues with some images. If the server blocks these requests, we won’t be able to access the headers directly. Let me know if you want to explore this further, or if you have any other ideas!

**Comment by Anush008** [comment]
> Thanks for pointing this out! Currently, the Web UI indeed looks for image URLs containing extensions like .jpg|jpeg|png|webp|gif|svg within the string, not just at the end.

Right. Got it. That would still miss URLs like https://www.bothellwa.gov/ImageRepository/Document?documentID=8024

> Regarding the idea of checking the content type via a HEAD request to confirm it's an image – that’s a solid approach, but it might run into CORS issues with some images. If the server blocks these requests, 

We don't have to render such images if a server blocks them.

**Comment by jayantpranjal0** [comment]
Is this issue still open to work upon?
